products/jbrowse-cli/tsconfig.json

```diff
@@ -7,9 +7,9 @@
     "outDir": "lib",
     "rootDir": "src",
     "strict": true,
-    "allowSyntheticDefaultImports": true,
     "noEmit": false,
-    "target": "es2017"
+    "target": "es2017",
+    "esModuleInterop": true
```

Comment:
`esModuleInterop` is already true in the parent tsconfig.json, so we don't need to specify it here.

Reply:
Removing esModuleInterop, for me, makes path undefined

Reply:
Note that I am just running bin/run admin-server in the products/jbrowse-cli directory and console.log(path)

Reply:
Ok, it looks like oclif doesn't follow the `extends`. My mistake.